utils.py

```diff
@@ -83,7 +83,7 @@ def get_file_contents(filepath, raw_bytes=False, mode='r'):
         *. Either the raw_bytes stream if the flag is set or the
             decoded stream in utf-8 format.
     """
-    with open(filepath, mode) as f:
+    with python_utils.open_file(filepath, mode) as f:
```

Comment:
Why this change, didn't it work with just `open`?

Reply:
@aks681 Actually, it was working fine with `open` but raising pylint error that's why I had to change it to `python_utils.open_file()`.